base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -2159,11 +2159,29 @@
         <type>PRESSURE_GAUGE</type>
         <path>nimrod_gas_panel.opi</path>
         <description>The OPI for the NIMROD Gas Panel</description>
-        <macros />
+        <macros/>
         <categories>
           <category>Gas and liquid handling systems</category>
         </categories>
       </value>
     </entry>
+    <entry>
+      <key>NIMROD Oerlikon Centre-One Pressure Gauge</key>
```

Comment:
I'm not sure this is unique to NIMROD - usually unless it's truly fixed (e.g. jaw sets or fully custom gas panels etc) we would just call this "Oerlikon Centre-One"

Reply:
Amended to not mention nimrod in record